casepro/cases/views.py

```diff
@@ -69,7 +69,8 @@ def get_context_data(self, **kwargs):
             # angular app requires context data in JSON format
             context['context_data_json'] = json_encode({
                 'all_labels': [l.as_json() for l in labels],
-                'fields': [f.as_json() for f in fields]
+                'fields': [f.as_json() for f in fields],
+                'message_id': case.initial_message.backend_id
```

Comment:
Not getting why is needed here - can't this just be added to the case JSON representation?